addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -162,8 +164,8 @@ class ComposerTextInput extends Component {
      * @private
      */
     _updateHeight() {
-        this._textareaRef.el.style.height = "0px";
-        this._textareaRef.el.style.height = (this._textareaRef.el.scrollHeight) + "px";
+        this._textareaDummyRef.el.style.height = "0px";
```

Comment:
Do we even need this line then? If height is already set at 0 in the XML that's all we need.